modules/swiper/module.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
Maybe it can be a bit cleaner if not all these functions are static functions?
WDYT?